src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -761,7 +760,7 @@ public class StripePaymentService : IPaymentService
             {
                 try
                 {
-                    if (!isPm5864DollarThresholdEnabled && !invoiceNow)
+                    if (!invoiceNow)
```

Comment:
This block was previously only accessible when the feature flag was disabled. With the flag enabled, this block is being skipped in production. Is that intended? The logic is effectively changed by deleting `!isPm5864DollarThresholdEnabled &&`.

Verify that the original logic `if (!isPm5864DollarThresholdEnabled && !invoiceNow) { ... }` was intended or not. It's possible that there was a bug before and your changes in this PR fixes that bug.